tests/perf-test/SidebarUtils.perf-test.ts

```diff
@@ -24,60 +27,60 @@ afterEach(() => {
     Onyx.clear();
 });
 
-const getMockedReportsMap = (length = 100) => {
-    const mockReports = Array.from({length}, (v, i) => {
-        const reportID = i + 1;
-        const participants = [1, 2];
-        const reportKey = `${ONYXKEYS.COLLECTION.REPORT}${reportID}`;
-        const report = LHNTestUtils.getFakeReportWithPolicy(participants, 1, true);
+const getMockedReports = (length = 500) =>
+    createCollection<Report>(
+        (item) => `report_${item.reportID}`,
```

Comment:
I think you should prefix using a predefined key (in case it changes anytime)

Reply:
done